locator/abstract_replication_strategy.cc

```diff
@@ -368,6 +368,14 @@ vnode_effective_replication_map::~vnode_effective_replication_map() {
     }
 }
 
+effective_replication_map::effective_replication_map(abstract_replication_strategy::ptr_type rs,
```

Comment:
In the commit message `s/token_effective_replication_map/vnode_effective_replication_map/`

Reply:
Fixed.